PerimeterXModule/PxModule.cs

```diff
@@ -63,6 +63,7 @@ public class PxModule : IHttpModule
 		private readonly StringCollection fileExtWhitelist;
 		private readonly StringCollection routesWhitelist;
 		private readonly StringCollection useragentsWhitelist;
+		private readonly StringCollection blockSpecificRoutes;
```

Comment:
I'd consider renaming to `enforceSpecificRoutes`. Using 'block' in the name can be misleading.

Reply:
👍